### PR TITLE
[AN-5119] conversation list shows automatically when the orientation changes to landscape

### PR DESCRIPTION
**Pull request by makingthematrix**
#### APK
[Download build #8963](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8963/artifact/build/artifact/wire-dev-PR908-8963.apk)
[Download build #8964](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8964/artifact/build/artifact/wire-dev-PR908-8964.apk)
[Download build #8965](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8965/artifact/build/artifact/wire-dev-PR908-8965.apk)
[Download build #8973](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8973/artifact/build/artifact/wire-dev-PR908-8973.apk)
[Download build #8984](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8984/artifact/build/artifact/wire-dev-PR908-8984.apk)
[Download build #8985](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8985/artifact/build/artifact/wire-dev-PR908-8985.apk)
[Download build #8987](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8987/artifact/build/artifact/wire-dev-PR908-8987.apk)
[Download build #8988](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8988/artifact/build/artifact/wire-dev-PR908-8988.apk)

### CODE REVIEW

**Comment by makingthematrix** [comment]
build again

**Comment by mykola-mokhnach** [comment]
Current build randomly forces the app to close right after it is  starte. This happens quite frequently if I restart the app being in landscape mode.

**Comment by mykola-mokhnach** [comment]
Only one issue is left there - the "hamburger" button behaves strange (disappears or closes the app after tap) if I switch to pending conversation view (incoming or outgoing) and then back to "normal" one

**Comment by mykola-mokhnach** [comment]
It anyway works better now than before. there is only issue with switching to pending conversations and back, but I'll create a separate issue for that

File: app/src/main/java/com/waz/zclient/pages/main/RootFragment.java [hunk 1]
```diff
@@ -689,9 +690,10 @@ public void run() {
         slidingPaneLayout.postDelayed(new Runnable() {
             @Override
             public void run() {
-                if (slidingPaneLayout == null) {
+                if (slidingPaneLayout == null || ViewUtils.isInLandscape(getActivity())) {
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/RootFragment.java]: Might be good to check if `getActivity` isn't null

File: app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java [hunk 2]
```diff
@@ -270,6 +274,33 @@ public void updated(SyncIndicator syncIndicator) {
...
+        if (isInLandscape != this.isInLandscape) {
+            this.isInLandscape = isInLandscape;
+            boolean conversationListVisible = (getControllerFactory().getNavigationController().getCurrentPage() == Page.CONVERSATION_LIST);
+
+            if (isInLandscape && !conversationListVisible) {
+                int duration = getResources().getInteger(R.integer.framework_animation_duration_short);
+                // post to give the RootFragment the chance to drive its animations first
+                new Handler().postDelayed(new Runnable() {
+                    @Override public void run() {
+                        getActivity().onBackPressed();
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java]: Make sure `getActivity` is not null

File: app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java [hunk 3]
```diff
@@ -270,6 +274,33 @@ public void updated(SyncIndicator syncIndicator) {
         }
     };
 
+    private Boolean isInLandscape = null;
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java]: Move variable to top of class

**Pull request merged**